cli/command/image/trust.go

```diff
@@ -97,16 +97,14 @@ func PushTrustedReference(streams command.Streams, repoInfo *registry.Repository
 	}
 
 	if target == nil {
-		fmt.Fprintln(streams.Out(), "No targets found, please provide a specific tag in order to sign it")
-		return nil
```

Comment:
Note this is now an error, so the cli will exit with non-zero. Since the push was not  actually completely it seems like this should be an error.